docs/building-apps/example-application-tutorial/payment.mdx

```diff
@@ -0,0 +1,367 @@
+---
+title: Payment
+sidebar_position: 40
+---
+
+A payment operation sends an amount in a specific asset (XLM or non-XLM) to a destination account. With a basic payment operation, the asset sent is the same as the asset received. BasicPay also allows for path payments (where the asset sent is different than the asset received), which we’ll talk about in the next section.
+
+## User experience
+
+In our BasicPay application, the user will navigate to the Payments page where can either select a user from their contacts or input the public key of a destination address with a specified asset they’d like to send along with the amount of the asset.
+
+![payment](/assets/payment.png)
+
+The user clicks the "Confirm Transaction" button. If the destination account exists and is properly funded with XLM, this will trigger a Transaction Preview where they can view the transaction details.
+
+All Stellar transactions require a small fee to make it to the ledger. Read more in our [Fees, Surge Pricing, and Fee Strategies section](../../encyclopedia/fees-surge-pricing-fee-strategies).
+
+In BasicPay, we’ve set it up so that the user always pays a static fee of 100,000 [stroops](../../fundamentals-and-concepts/stellar-data-structures/assets#amount-precision) (one stroop equals 0.0000001 XLM) per operation. Alternatively, you can add a feature to your application that allows the user to set their own fee.
+
+![payment](/assets/fees.png)
+
+The user then inputs their pincode and clicks the "Confirm" button, which signs and submits the transaction to the ledger.
+
+:::info
+
+An asset is displayed as an asset code and issuer address. Learn more in our [Assets section](https://developers.stellar.org/docs/fundamentals-and-concepts/stellar-data-structures/assets).
+
+:::
+
+## Code implementation
+
+### The `/dashboard/send` page
+
+{/* TODO: Bri, is this too much from this code sample? Should we just cut ALL of the HTML out? */}
```

Comment:
Agreed. Especially since we'll have the direct link to the source file underneath, it makes the most sense.

Even the HTML I had put into the code blocks had to be customized and pared down quite a bit. So, I think users would be better served having the _whole_ context.